content/bd/0001-intro-dbms.md

```diff
@@ -228,33 +228,33 @@ representa um único aluno.
 
 | `ist_id`   | `student_name` | `birthday` |
 | ---------- | -------------- | ---------- |
-| ist1123456 | Diogo          | 2002-03-06 |
-| ist1123234 | Rafa           | 2002-08-05 |
-| ist1124453 | Tomás          | 2002-12-18 |
-| ist1123534 | Diogo          | 2001-09-16 |
-| ist1123532 | Diogo          | 2002-04-27 |
+| ist123456 | Diogo          | 2002-03-06 |
+| ist123234 | Rafa           | 2002-08-05 |
+| ist124453 | Tomás          | 2002-12-18 |
+| ist123534 | Diogo          | 2001-09-16 |
+| ist123532 | Diogo          | 2002-04-27 |
 
 :::
 
-Neste exemplo faria também sentido impor restrições de integridade.
+Neste exemplo, faria também sentido impor restrições de integridade.
 Neste caso, a restrição mais óbvia é garantir que não existem dois alunos diferentes
 com o mesmo `ist_id`. No entanto, podem existir alunos com o mesmo nome.  
-Mais à frente veremos como podemos indicar ao SGBD para seguir estas regras,
+Mais à frente, veremos como podemos indicar ao SGBD para seguir estas regras,
 de forma a assegurar a integridade dos dados.
 
 ## Arquitetura de SGBDs
 
-Para falarmos com os SGBDs, temos de usar uma linguagem que eles compriendam de modo a
-expressar de forma clara as nossas intenções. No caso do modelo relacional, é comummente
+Para falarmos com os SGBDs, temos de usar uma linguagem que eles compreendam de modo a
+expressar de forma clara as nossas intenções. No caso do modelo relacional, é geralmente
 utilizado o [**SQL**](color:orange) (Structured Query Language).
 
-Na imagem abaixo podemos ver o esquema que representa a estrutura todo o fluxo de execução
+Na imagem abaixo, podemos ver o esquema que representa a estrutura de todo o fluxo de execução
 entre a aplicação e a base de dados, passando pelo SGBD.
 
 ![Esquema que representa a arquitetura de um SGBD](./assets/0001-dbms-architecture.svg#dark=3)
 
-Quando executamos uma _query_ (interrogação), esta é enviada ao
+Quando executamos uma _query_ (interrogação), esta é enviada à
```

Comment:
```suggestion
Quando executamos uma _query_ (interrogação), esta é enviada ao
```
`Engine` (= `motor`) is arguably a masculine word in Portuguese. Think `pesquisei no [motor de busca] Google]` (vs `a [empresa] Google despediu 100 pessoas`), `a configuração do nginx`, etc.)